great_expectations/profiler/profiler.py

```diff
@@ -60,12 +59,12 @@ def __init__(
                         )
                     )
 
-                configuration_builders = []
+                expectation_configuration_builders = []
```

Comment:
As per our conversation yesterday, I wonder whether it makes sense to leave these as `configuration_builders` to keep things generic, even if we know that for the immediate future the only configurations will be ExpectationConfigurations. This might be a good question for James during today's design review.

Reply:
@talagluck @anthonyburdi I would prefer to leave it as `expectation_configuration_builders` (i.e., "specific", rather than general), until we learn of more general requirements.  This is because, while we currently are thinking that other types of the `Configuration` may be extending the base `Configuration` class, it is quite possible that their use cases will be sufficiently different to warrant that the various configuration classes are the siblings of one another.  Please let me know whether or not you agree with this proposal.  Thank you.

Reply:
My (loosely-held) opinion is to disagree, though that is partially because James initially called these `configuration_builders` rather than `expectation_configuration_builders`, and I suspect that this might have been the reason. I think that discussing this today during the design review would help to sway me one way or the other.

Reply:
That was my take @alexsherstinsky but point well taken @talagluck. Let's keep it as an open question for now.

Reply:
How is this for an open question @talagluck ? `Are there other types of Configuration besides ExpectationConfiguration that a Profiler should output?`

Reply:
@talagluck @anthonyburdi Please bear in mind that any generalization up from `ExpectationConfiguration` will imply a refactor, including new `type` / `Union` declarations.

Reply:
Resolved to use `ExpectationConfiguration`.